test/interaction/AmbiguousConstructor.in

```diff
@@ -0,0 +1 @@
+top_command (cmd_load currentFile [])
```

Comment:
Tests that do no interaction, just trigger an error, should be placed in `test/Fail`.
You do not need an `.in` file, just the `.agda` file and the `.err` file (which is generated on the first run of `make fail`).